pcsx2/DEV9/ATA/ATA_Info.cpp

```diff
@@ -230,7 +238,12 @@ void ATA::CreateHDDinfo(u64 sizeSectors)
 	 * bit 13: FLUSH CACHE EXT
 	 * bit 14: 1
 	 */
-	WriteUInt16(identifyData, &index, static_cast<u16>((1 << 14) | (1 << 13) | (1 << 12) /*| (1 << 8)*/ | ((lba48Supported ? 1 : 0) << 10))); //word 83
+	WriteUInt16(identifyData, &index, ( 	//word 83
+		(lba48Supported << 10) | 			//user defined
+		(1 << 12) |							//implemented
+		(1 << 13) |							//implemented
+		(1 << 14) 							//Fixed
```

Comment:
```suggestion
		(1 << 14) 							//Always one
```

I've used `//Fixed` to note if a field is fixed of variable in the Command set/feature enabled words

Reply:
I see, was under impression, that `fixed` means always one or always zero.

Reply:
Yeah, I perhaps wasn't clear on that when I wrote the comments